src/index.html

```diff
@@ -184,6 +185,7 @@ <h2 i18n="best-waifu-ever">The best Waifu. <b>Ever.</b></h2>
 							<option disabled title="Spanish" value="es" lang="es">Español</option>
 							<option disabled title="Estonian" value="et" lang="et">eesti</option>
 							<option disabled title="French" value="fr" lang="fr">Français</option>
+							<option disabled title="Bahasa" value="id" lang="id">Bahasa</option>
```

Comment:
Sorry, I was half-awake when made this translation and I thought it was sorted by value so I put it  just below French XD
I'll put it at the top since its start with B